src/egon/data/db.py

```diff
@@ -16,7 +17,8 @@ def credentials():
         Complete DB connection information
     """
     # Read database configuration from docker-compose.yml
-    package_path = egon.data.__path__[0]
+    #package_path = egon.data.__path__[0]
+    package_path = files('egon.data')
```

Comment:
When the package has been imported, you can also just write `files(egon.data)`, guarding against typos.